src/plugins/intel_cpu/src/nodes/convert.cpp

```diff
@@ -126,7 +126,20 @@ void Convert::initSupportedPrimitiveDescriptors() {
         config.outConfs.push_back(dataConfigOut);
 
         auto creators = BlockedDescCreator::getCommonCreators();
-        auto range = BlockedDescCreator::makeFilteredRange(creators, insShape.getRank());
+
+        // As long as convert is placed right before the output, only planar layout makes sense since the output tensor
+        // is always in a planar layout (ngraph limitation), so there is no reason to convert in nspc (per Chanel
+        // layout) and reorder to the planar one.
```

Comment:
```suggestion
        // As long as convert is placed right before the output, only planar layout makes sense since the output tensor
        // is always in a planar layout (ngraph limitation), so there is no reason to convert in any other layout.
```

Reply:
done